base_multicompany_submit/models/multicompany_submit_mixin.py

```diff
@@ -0,0 +1,51 @@
+#  Copyright (c) Akretion 2020
+#  License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
+
+from odoo import api, fields, models
+
+
+class BaseMulticompanyMixin(models.AbstractModel):
+    _name = "base.multicompany.mixin"
```

Comment:
It doesn't define what for is the mixin. You could remove `base` and replace by a meaningful name ? submittal ?

Reply:
Thanks. Renamed to multicompany.submit.mixin